tests/framework/db/sqlite/SchemaTest.php

```
@@ -59,6 +65,205 @@ public function testCompositeFk(): void
         $this->assertEquals('item_id', $table->foreignKeys[0]['item_id']);
     }
 
+    public function testCurrentTimestampLowercaseDefaultValue(): void
+    {
+        $db = $this->getConnection(false);
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)